Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Include-Server-Commands](https://hosted.weblate.org/projects/jamulus/include-server-commands/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jamulus/-/1-index/horizontal-auto.svg)

ann0see commented on _translator-files/po/it/Installation-for-Windows.po:
```diff
@@ -1,31 +1,31 @@
 #, no-wrap
 msgid "wiki"
-msgstr ""
+msgstr "guida"
```
@dzpex please don't translate this. (and the related ones)

ignotus666 commented on _translator-files/po/it/Getting-Started.po:
```diff
@@ -1,53 +1,53 @@
 #, no-wrap
 msgid "/wiki/Getting-Started"
-msgstr ""
+msgstr "/guida/Primi-Passi"
```
This shouldn't be translated either.

ignotus666 commented on _translator-files/po/nb_NO/Client-Troubleshooting.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
@comradekingu please use the full language code here (nb_NO).

ann0see replied:
```suggestion
msgstr "nb_NO"
```

comradekingu replied:
nb_NO now in https://hosted.weblate.org/translate/jamulus/client-troubleshooting/nb_NO/?checksum=cc3a3b8aeba83df4

ignotus666 commented on _translator-files/po/nb_NO/Getting-Started.po:
```diff
@@ -141,18 +141,22 @@ msgstr ""
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"  <img src=\"{% include img/nb-screenshots/settings-profile.inc %}\" loading=\"lazy\" alt=\"Skjermavbildning\" style=\"border: 5px solid grey;\">\n"
```
"Screenshot" ("Skjermavbildning") shouldn't be translated - just the 'figcaption'.

comradekingu replied:
Is there a reason why? SEO and assumption people with screen-readers want Norwegian Bokmål (as opposed to English) when selecting the nb_NO locale is why I translated it.
If you want it untranslated, making it a placeholder works, because people will keep doing this.

ignotus666 replied:
I may have been mistaken in assuming it shouldn't be translated.

ignotus666 commented on _translator-files/po/pt_BR/Getting-Started.po:
```diff
@@ -160,7 +159,7 @@ msgid ""
 msgstr ""
 "<figure>\n"
-"  <img src=\"{% include img/en-screenshots/settings-profile.inc %}\" loading=\"lazy\" alt=\"Screenshot\" style=\"border: 5px solid grey;\">\n"
+"  <img src=\"{% include img/en-screenshots/settings-profile.inc %}\" loading=\"lazy\" alt=\"captura de tela\" style=\"border: 5px solid grey;\">\n"
```
Same again, don't translate "Screenshot".

ann0see commented on _translator-files/po/de/Getting-Started.po:
```diff
@@ -160,7 +159,7 @@ msgid ""
 msgstr ""
 "<figure>\n"
-"  <img src=\"{% include img/de-screenshots/settings-profile.inc %}\" loading=\"lazy\" alt=\"Screenshot\" style=\"border: 5px solid grey;\">\n"
+"  <img src=\"{% include img/de-screenshots/settings-profile.inc %}\" loading=\"lazy\" alt=\"Bildschirmkopie\" style=\"border: 5px solid grey;\">\n"
```
No. This should either stay or be "Bildschirmfoto".

ann0see replied:
```suggestion
"  <img src=\"{% include img/de-screenshots/settings-profile.inc %}\" loading=\"lazy\" alt=\"Bildschirmfoto\" style=\"border: 5px solid grey;\">\n"
```

ann0see commented on _translator-files/po/it/Command-Line-Options.po:
```diff
@@ -1,31 +1,31 @@
 #, no-wrap
 msgid "wiki"
-msgstr ""
+msgstr "guida"
```
Probably worth a bulk fix and a label on weblate

comradekingu replied:
Went to fix that, but "kingu" is not/no longer(?) an admin.
https://hosted.weblate.org/translate/jamulus/client-troubleshooting/en/?checksum=d9c67f8154384e99 is the string anyway.

ignotus666 replied:
If you change it in one doc, the correction will spread to all others in that language.

ignotus666 replied:
Just changed it to verify - it does automatically spread.

ann0see reviewed (changes requested):
Still needs some work... but we'll release the website today evening.

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: Jamulus/Directories
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/
Translation: Jamulus/Tips-Tricks-More

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/
Translation: Jamulus/Tips-Tricks-More

ann0see commented on _translator-files/po/it/Getting-Started.po:
```diff
@@ -1,53 +1,54 @@
 #, no-wrap
 msgid "/wiki/Getting-Started"
-msgstr ""
+msgstr "/guida/Primi-Passi"
```
```suggestion
msgstr "/wiki/Getting-Started"
```

ann0see commented on _translator-files/po/it/navigation.po:
```diff
@@ -60,7 +61,7 @@ msgstr "/wiki/Installation-for-Linux"
 #, no-wrap
 msgid "/wiki/Getting-Started"
-msgstr "/wiki/Getting-Started"
+msgstr "/guida/Primi-Passi"
```
```suggestion
msgstr "/wiki/Getting-Started"
```

ann0see commented on _translator-files/po/nb_NO/Command-Line-Options.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Contribution.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Directories.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/FAQ.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Getting-Started.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Installation-for-Android.po:
```diff
@@ -1,68 +1,71 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Installation-for-Linux.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Installation-for-Macintosh.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Installation-for-Windows.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Installation-for-iOS.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Privacy-Statement.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/QOS-Windows.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Running-a-Server.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Server-Bandwidth.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Server-Troubleshooting.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Software-Manual.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see commented on _translator-files/po/nb_NO/Tips-Tricks-More.po:
```diff
@@ -1,31 +1,31 @@
+#, fuzzy, no-wrap
 msgid "en"
-msgstr ""
+msgstr "nb"
```
```suggestion
msgstr "nb_NO"
```

ann0see committed:
Add final fixes
